### ISSUE TITLE
LicenseMetadata editor

### ISSUE THREAD

**Issue by clairernovotny** [NuGetPackageExplorer/NuGetPackageExplorer#528]
NuGet 4.9 adds a `LicenseMetadata`. We need an editor for it.

**Comment by onovotny** [comment]
Yes, that's what I mean. Right now it displays the data but does not have any controls for setting/editing it outside of the XML nuspec view.
